app/models/anime.rb

```diff
@@ -111,10 +111,8 @@ def season_year
   alias_method :year_changed?, :season_changed?
 
   def update_unit_count_guess(guess)
-    return if episode_count
-    return if episode_count_guess && episode_count_guess > guess
+    return if episode_count || episode_count_guess != guess
```

Comment:
Won't `episode_count_guess != guess` always be the case?  I mean, isn't the whole point of this method to modify it?

So like, if `episode_count_guess` isn't set, it'll be `nil != guess`, and if it's already set this won't update it

Reply:
whoops, yeah